integration/update_array_compat_test.go

```diff
@@ -120,3 +120,47 @@ func TestUpdateArrayCompatPush(t *testing.T) {
 
 	testUpdateCompat(t, testCases)
 }
+
+func TestUpdateArrayCompatAddToSet(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]updateCompatTestCase{
```

Comment:
I know we usually don't use filters in updates, but can we use one case filtered by `_id` where we try to `addToSet` the value that is already in the set? Or if it's covered by one of the current test cases, let's add a comment that clarifies which document (with which `_id`) covers it.

Reply:
Added a comment.